pkg/agent/wireguard/client_linux.go

```diff
@@ -94,6 +97,27 @@ func (client *client) Init() error {
 	if err := netlink.LinkSetUp(link); err != nil {
 		return err
 	}
+	// Configure the IP addresses same as Antrea gateway so iptables MASQUERADE target will select it as source address.
+	// It's necessary to make service traffic requiring SNAT accepted by peer Node and to make their response routed
```

Comment:
service -> Service

Reply:
Could we add the types of Service traffic here too? It helps me to understand.

Reply:
Does "service traffic requiring SNAT" explain it? If listing types, I need to list "host-to-ClusterIP, host-to-NodePort, external-to-NodePort, host-to-externalIP, external-to-externalIP", maybe more..

Reply:
How about "external IP", "NodePort", and "host to ClusterIP"? Or just "Service traffic from host network" (I know there is externalTrafficPolicy but probably no need to go all details).

Reply:
Done, thanks